tests/filecheck/dialects/asl/constant_ops.mlir

```diff
@@ -1,17 +1,20 @@
 // RUN: asl-opt %s | asl-opt %s | filecheck %s
 
 builtin.module {
-    %true = asl.constant_bool true {attr_dict}
-    %false = asl.constant_bool false {attr_dict}
+    %true = asl.constant_bool true {"attr_dict"}
```

Comment:
This is my bad, I switched from a commit to just "main" in xDSL, so I have a feeling that there's drift in our versions, we recently merged a change that avoide the quotes when their content is a valid symbol, so the version on the left in this diff is actually the more up to date spelling.

Reply:
Cool - I rebased and this was not needed.